### DISCUSSION TITLE
W11 - Add "Quick Launch" to Taskbar?...

### DISCUSSION THREAD

**Discussion by stani63** [ramensoftware/windhawk-mods#395] [General]
Add "Quick Launch" to Taskbar in Windows 11.
[Would it be possible](https://discord.com/channels/923944342991818753/923954164583759873/1168207970010669076)?

**Comment by m417z** [comment]
As I wrote in Discord:

A possible solution is to use the "Disable grouping on the taskbar" mod with the "Keep pinned items separated" option set to true. Right now, there's no way to use this option without having the main functionality to also disable grouping. I planned on adding this as an option, let me know if it's relevant for you.

Would that work for you?

Here's a screenshot of the 7+ Taskbar Tweaker help file for a similar option, which shows how it looks:

![image](https://github.com/ramensoftware/windhawk/assets/4129781/ca32136f-74a7-437b-88ef-eba42bd08dbf)

Note that implementing the real Quick Launch for the Windows 11 taskbar isn't trivial, as the functionality is simply not there.

**Reply by stani63** [reply]
Hi, 

if it worked with left mouse click, that would probably be it.

**Comment by stani63** [comment]
Will you please do it?

**Reply by m417z** [reply]
I'll look into implementing it when I have some spare time.

**Reply by m417z** [reply]
I added a new option to the **Disable grouping on the taskbar** mod, "Pinned items remain in place, group running instances", which keeps pinned items separated from running instances, while keeping the running instances grouped together.

**Reply by stani63** [reply]
Good work! :-)
